docs/presto-admin-commands.rst

```diff
@@ -489,14 +489,14 @@ server upgrade
 **************
 ::
 
-    presto-admin server upgrade local_package_path [local_config_dir]
+    presto-admin server upgrade new-rpm [local_config_dir]
```

Comment:
I think the whole issue that John had was with the word path -- it made him think that he should specify the directory